services/api/sample.env

```diff
@@ -14,4 +14,8 @@ CSM_EMAIL_APP_USERNAME=
 CSM_EMAIL_APP_PASSWORD=
 CSM_EMAILS_TO_SEND_TO=
 CSM_TARGET_SMTP_SERVER_ADDRESS=
-CSM_TARGET_SMTP_SERVER_PORT=
\ No newline at end of file
+CSM_TARGET_SMTP_SERVER_PORT=
+
+# Error Email Contact Configuration
+CLOUD_RUN_LOGS_LINK=https://console.cloud.google.com/run/detail/us-central1/rsu-manager-cloud-run-api/logs?authuser=1&project=cdot-oim-cv-dev

```

Comment:
Since this is the sample.env should we remove these values?

Reply:
I go both ways. Sometimes it is nice if the sample.env has samples of non-secret values. But that can be confusing to some people, because they will have to change it still. I will remove it